.storybook/main.js

```diff
@@ -23,27 +25,48 @@ module.exports = {
       return data;
     });
 
+    /**
+     * For development, leave the default 'cheap-module-source-map', as it's faster and works
+     * For the build, using the default does not work correctly, but this option appears to
```

Comment:
Nit: Should these sentences end in periods? At first glance I thought this was a run-on sentence.